cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ReplicaDistributionGoal.java

```diff
@@ -101,8 +101,7 @@ protected void updateGoalState(ClusterModel clusterModel, OptimizationOptions op
       ProvisionRecommendation recommendation = new ProvisionRecommendation.Builder(ProvisionStatus.OVER_PROVISIONED)
           .numBrokers(numBrokersToDrop).build();
       _provisionResponse = new ProvisionResponse(ProvisionStatus.OVER_PROVISIONED, recommendation, name());
-    } else if (_succeeded) {
-      // The cluster is not overprovisioned and all brokers are within the upper and lower balance limits.
+    } else {
```

Comment:
Did we check the other goals, which set the _provisionResponse to `OVER_PROVISIONED` for the same bug?

Reply:
Yes, others will not have the same issue.